src/store/chatMessages.js

```diff
@@ -0,0 +1,13 @@
+export default (state = [], action) => {
+    switch (action.type) {
+    case 'GET_MESSAGES':
+        return action.messages;
+    case 'SEND_MESSAGE':
```

Comment:
I would honestly call it something like "MESSAGE_RECEIVED" or something like that, cause I understand it's dispatched after the socket.io gets the CHAT_MESSAGE and emits appropriate event. so to our store it looks like we are always getting the messages, not sending them